fpnge.cc

```diff
@@ -736,33 +743,21 @@ FORCE_INLINE void WriteBits(MIVEC nbits, MIVEC bits_lo, MIVEC bits_hi,
   constexpr uint8_t kPerm[] = {0, 1, 2, 3};
 #endif
 
-  // call to WriteBits manually inlined: compiler assumes aliasing may happen.
```

Comment:
I assume this is fine because of the __restrict, right?

Reply:
From testing and looking at the assembly, it appears so.